scripts/gen_funcs.py

```diff
@@ -0,0 +1,57 @@
+# Python script to generate Vim Script syntax definitions.
+#
+# Copyright (C) 2024    Luca Zeuch, Lucas Ritzdorf
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+
+import os
+
+def gen_code(file):
```

Comment:
```suggestion
def gen_code(funcs):
```
and change `gen_code` to take a list of functions? (So the caller should read the `lytfs` file and split lines.)